app/views/tasks/edit.html.haml

```diff
@@ -1,32 +1,2 @@
-%h1 Update the Task
-= form_with(model: @task, data: { turbo: false }, url: task_path(@task)) do |f|
-  %p Title
-  = f.text_field :title
-
-  %p Description
-  = f.text_field :description
-
-  %p Due to
-  = f.date_field :due_date
-
-  %p Status
-  = f.check_box :status, { checked: @task }, :completed, :active
-  = f.label "Completed"
-
-  %p "Priority"
-  - Task.priorities.each do |key, value| 
-    = f.radio_button :priority, key
-    = f.label "priority_#{key}", key.to_s.humanize
-
-  %p Files
-  - @task.images.each do |image|
-    = f.hidden_field :images, multiple: true, value: image.signed_id
-
-  = f.file_field :images, multiple: true
-
-  = f.submit "Update Task"
-
-- if @task.errors.any?
-  - @task.errors.full_messages.each do |message|
-    %p= message
-    
\ No newline at end of file
+%h3.mt-5.mb-5.text-center Update the Task
+= render "form", obj: @task
```

Comment:
No core differences but better to name local var same as model. For ex. here `task: @task`